pyvrp/_ProblemData.pyi

```diff
@@ -138,24 +136,6 @@ class ProblemData:
         int
             Travel duration between the given clients.
         """
-    def distance_matrix(self) -> Matrix:
-        """
-        Returns the travel distance matrix used for distance computations.
-
-        Returns
-        -------
-        Matrix
-            Travel distance matrix.
-        """
-    def duration_matrix(self) -> Matrix:
-        """
-        Returns the travel duration matrix used for duration computations.
-
-        Returns
-        -------
-        Matrix
-            Travel duration matrix.
```

Comment:
I have removed these since it is rather tricky to switch from dist/dur types (internal to C++) back to int/floats all the time. We might find a better solution for this once we switch to numpy-like matrices, but for now I don't have one.